Dockerfile

```diff
@@ -40,6 +40,9 @@ RUN dnf install -y python3 python3-requests && \
     dnf clean all && \
     rm -rf /var/cache/{yum,dnf}/*
 
+# install vendor driver libs
+RUN /usr/bin/pip3 install python-ibmcclient>=0.2.3
```

Comment:
please we should not install anything using pip but only rely on packages as for other requirements

Reply:
hi @elfosardo , `python-ibmcclient` is available now at: https://trunk.rdoproject.org/centos8/component/baremetal/45/eb/45ebd10ea96a436f0ba6bd06dd393df26d333606_168399e5/